docs/DeveloperGuide.md

```diff
@@ -283,9 +284,59 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. GradPad shows an error message.
+
+      Use case resumes at step 2.
+      
+**Use case : Add a Module**
+
+**MSS**
+
+1. User requests to list Modules
+2. GradPad shows a list of Modules
```

Comment:
Actually does the user need to list all modules first before adding a new one ah?